### PR TITLE
[@mantine/core] Tooltip: Add the `radius` prop

### PR DESCRIPTION
**Pull request by jerebtw**
Fixes https://github.com/mantinedev/mantine/issues/560

### CODE REVIEW

**Comment by rtivital** [comment]
How does it solve the bug with arrow discussed in the issue?

**Comment by jerebtw** [comment]
We still need to find a fix for that problem

**Comment by rtivital** [comment]
I do not think that we should provide a feature that does not work, so this PR can be merged only with a fix

**Comment by jerebtw** [comment]
Okay I will think about a fix

File: src/mantine-core/src/components/Popper/Popper.tsx [hunk 1]
```diff
@@ -21,6 +21,9 @@ export interface SharedPopperProps {
   /** Arrow size in px */
   arrowSize?: number;
 
+  /** Arrow distance to the left/right * arrowSize */
```
Comment (jerebtw) [inline src/mantine-core/src/components/Popper/Popper.tsx:24]: @rtivital is that comment right?

**Comment by jerebtw** [comment]
> I do not think that we should provide a feature that does not work, so this PR can be merged only with a fix

I fixed the issue 👍🏻 I just moved the arrow a little

**Comment by rtivital** [comment]
Thanks, it works great!

**Pull request merged**